boot/src/main/java/com/zextras/carbonio/files/Boot.java

```diff
@@ -54,6 +56,9 @@ public void boot() {
       purgeService = injector.getInstance(PurgeService.class);
       purgeService.start();
 
+      messageBrokerManager = injector.getInstance(MessageBrokerManager.class);
+      messageBrokerManager.startAllConsumers();
+
```

Comment:
Can the MessageBrokerManager stop all the consumers gracefully when there is an exception or when the exection reaches the `finally` block?

Tips: Think about extending the `AutoClosable` interface in the `MessageBrokerManagerImpl`